modules/serverless-ossm-traffic-isolation-architecture.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: CONCEPT
+[id="serverless-ossm-traffic-isolation-architecture_context"]
+= High-level architecture
+
+The high-level architecture of {ServerlessProductShortName} traffic isolation provided by {SMProductShortName} consists of `AuthorizationPolicies` in the `knative-serving`, `knative-eventing` and the tenants namespaces, with all the components being part of the {SMProductShortName}. The injected {SMProductShortName} sidecars enforce those rules to isolate network traffic between tenants.
+
+[id="serverless-ossm-traffic-isolation-architecture-serving"]
+== Knative Serving
+
+Knative Serving adheres to the following architecture:
+
+image::multi-tenancy-service-mesh-serving.png[Knative Serving architecture]
+
+The following relationships apply:
+
+. Workloads in project `tenant-2` cannot call workloads in project `tenant-1` directly. Requests are rejected in the Istio proxies of `tenant-1` workloads.
+
+. Workloads in project `tenant-2` cannot call workloads in project `tenant-1` through ingress gateway. Requests are rejected in the Istio proxies of `tenant-1` workloads.
+
+. Workloads in project `tenant-2` cannot call workloads in project `tenant-1` through activator. Requests are rejected in the Istio proxy of the activator component. Since ingress gateway is allowed to call activator, this includes requests from project `tenant-2` through ingress gateway to activator.
```

Comment:
```suggestion
. Workloads in project `tenant-2` cannot call workloads in project `tenant-1` through activator. Requests are rejected in the Istio proxy of the activator component. Because ingress gateways are allowed to call the activator, this includes requests from project `tenant-2` through ingress gateway to the activator.
```
"Because" instead of "Since" is better for translation. Also "ingress gateway is allowed" singular sounded strange, and plain "activator" without a "the" sounds strange, but disregard both if they're intentional.